packages/module/src/ListManager/ListManager.tsx

```diff
@@ -114,41 +146,30 @@ const ListManager: FunctionComponent<ListManagerProps> = (
           }
         />
       </div>
-      <DragDropSort
-        variant="DataList"
-        items={currentColumns.map((column, index) => ({ id: column.key, content: 
-            <DataListItemRow>
-              <DataListCheck
-                data-testid={`column-check-${column.key}`}
-                isChecked={column.isSelected}
-                onChange={() => handleChange(column.key)}
-                isDisabled={column.isUntoggleable}
-                aria-labelledby={`${ouiaId}-column-${index}-label`}
-                ouiaId={`${ouiaId}-column-${index}-checkbox`}
-                id={`${ouiaId}-column-${index}-checkbox`}
-              />
-              <DataListItemCells
-                dataListCells={[
-                  <DataListCell key={column.key} data-ouia-component-id={`${ouiaId}-column-${index}-label`}>
-                    <label htmlFor={`${ouiaId}-column-${index}-checkbox`} id={`${ouiaId}-column-${index}-label`}>
-                      {column.title}
-                    </label>
-                  </DataListCell>
-                ]}
-              />
-            </DataListItemRow>
-        }))}
-        onDrop={onDrag}
-        overlayProps={{ isCompact: true }}
-      >
-        <Droppable 
-          items={currentColumns.map((column) => 
-            // eslint-disable-next-line no-console
-            ({ id: column.key, content: column.title })
-          )} 
-          wrapper={<DataList aria-label="Selected columns" isCompact data-ouia-component-id={`${ouiaId}-column-list`}/>}
-        />
-      </DragDropSort>
+      {enableDragDrop ? (
+        <DragDropSort
+          variant="DataList"
+          items={currentColumns.map((column, index) => ({ id: column.key, content: renderDataListItem(column, index) }))}
+          onDrop={onDrag}
+          overlayProps={{ isCompact: true }}
+        >
+          <Droppable
+            items={currentColumns.map((column) =>
+              // eslint-disable-next-line no-console
+              ({ id: column.key, content: column.title })
+            )}
+            wrapper={<DataList aria-label="Selected columns" isCompact data-ouia-component-id={`${ouiaId}-column-list`}/>}
```

Comment:
It's pre-existing prior to this PR so I'd be fine with a followup for it, but we should provide a prop for customizing the aria-label on the DataList here and line 165